schemas/code_signature.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: code_signature
+  title: Code Signature
+  group: 2
+  description: These fields contain information about binary code signatures.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - process
+      # - dll
```

Comment:
Let's merge this PR before DLL as well, and add this reuse of `code_signature` via the DLL PR (#679) as well.